test/src/main/java/org/apache/accumulo/test/ShellServerIT.java

```diff
@@ -1570,16 +1570,13 @@ public void listscans() throws Exception {
       SlowIterator.setSleepTime(cfg, 500);
       s.addScanIterator(cfg);
 
-      Thread thread = new Thread() {
-        @Override
-        public void run() {
-          try {
-            Iterators.size(s.iterator());
-          } catch (Exception ex) {
-            throw new RuntimeException(ex);
-          }
+      Thread thread = new Thread(() -> {
+        try {
+          Iterators.size(s.iterator());
+        } catch (Exception ex) {
+          throw new RuntimeException(ex);
```

Comment:
Does this propagate to the parent thread? Or is it lost? If lost, can we fix that?

Reply:
No checked exceptions were being thrown.